app/views/openid_connect/authorization/redirect.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title><%= t('saml_idp.shared.saml_post_binding.redirecting') %> | <%= APP_NAME %></title>
```

Comment:
> 1. since the `authorization/redirect.html.erb` and `logout/redirect.html.erb` are identical as far as I can tell, could we add like a `app/views/openid_connect/shared/redirect.html.erb` and unify?

Good call, done in 8ad0eccb8a

> 2. this i18n string still has `saml` in it, +1 to earlier comment to move to something shared, but don't consider this a blocking comment

Makes sense, moved it to `headings/` in 3d33141a0aa3e1b0b61a3929c25b23e1fdfdb97a